examples/chip-tool/commands/clusters/ColorControl/Commands.h

```diff
@@ -0,0 +1,273 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#ifndef __CHIPTOOL_COLORCONTROL_COMMANDS_H__
+#define __CHIPTOOL_COLORCONTROL_COMMANDS_H__
+
+#include "../../common/ModelCommand.h"
+
+class MoveToHue : public ModelCommand
+{
+public:
+    MoveToHue(const uint16_t clusterId) : ModelCommand("move-to-hue", clusterId) {}
+
+    size_t EncodeCommand(PacketBuffer * buffer, size_t bufferSize, uint16_t endPointId) override
```

Comment:
After the latest code namespace submit, System::PacketBuffer is not global anymore.

You may need to merge master and do some rebasing.

Reply:
Thankfully I do not need to change anything. Those are coming from https://github.com/project-chip/connectedhomeip/blob/master/examples/chip-tool/commands/common/Command.h#L65